web/src/playground_links_popover.ts

```diff
@@ -103,14 +114,15 @@ function register_click_handlers() {
                     url_template.expand({code: extracted_code}),
                 );
             } else {
+                const playground_store = new Map<RealmPlayground, string>();
```

Comment:
What I'd had in mind is this being a `Map<string, RealmPlaygroundWithURL>`

Reply:
`string` ?? I think using playground `id`( type `number`) as key would be simpler as you referred in [above comment](https://github.com/zulip/zulip/pull/28386#discussion_r1486585997) ie `Map<PlaygroundId, RealmPlaygroundWithURL>` passed as argument.

Reply:
Yeah that'd be better.